routes/survey.js

```diff
@@ -46,18 +46,18 @@ function prefixObj(row, prefix) {
   }
 }
 
-router.get('/:id/', async (req, res) => {
+router.get('/:survey/:id/', async (req, res) => {
   const { app } = req;
-  let { id: _id } = req.params;
+  const { id: _id, survey } = req.params;
   const { compareTo = '0' } = req.query;
 
-  let [ idPrefix, selectionId ] = _id.split('_');
+  let [idPrefix, selectionId] = _id.split('_'); // eslint-disable-line
```

Comment:
wait, why prefer using `// eslint-disable-line` here?

Reply:
I actually added that because eslint was complaining about `idPrefix` being a `let` because it doesn't get reassigned, but I'll definitley put the spaces back

Reply:
ohhh gotcha